lib/ansible/modules/cloud/docker/docker_volume.py

```diff
@@ -22,7 +22,7 @@
                     'supported_by': 'community'}
 
 
-DOCUMENTATION = u'''
```

Comment:
Why did you remove the `u`? My name contains a non-ASCII character and thus may not display correctly on Python 2 unless the docstring is unicode, yes?